README.md

```diff
@@ -3,7 +3,7 @@
 
 This module provides cartridge roles implementing of a distributed queue compatible with [Tarantool queue](https://github.com/tarantool/queue) (*fifiottl driver*)
 
-## Installing
+## Installing (as a dependency for your cartridge app)
```

Comment:
Is there another way?

Reply:
`as a service`, okay

But then it's better to write something like: `Usage in a cartridge application`

Reply:
Fixed